### DISCUSSION TITLE
opencascade and oce

### DISCUSSION THREAD

**Discussion by revelator** [msys2/MINGW-packages#11577] [Q&A] [answered]
i was wondering if these dont conflict ? if so the PKGBUILD should probably be updated or was there some change that allow these two to coexist.

**Comment by Biswa96** [comment, marked as answer]
That conflict was added here https://github.com/msys2/MINGW-packages/pull/11508 by @stahta01

**Reply by revelator** [reply]
thx though i wonder can gmsh be used with oce instead of opencascade ? else i would have to remove this as well for kicad build.

**Reply by stahta01** [reply]
I tried to install both and got conflicting files.

Tim S.

**Reply by stahta01** [reply]
My upgrade for KiCAD is going to be using opencascade in 5.1.12 because 6.0.4 no longer allows OCE to be used.

I am testing my 6.0.4 code changes right now, but, I have an old Intel GPU card that has issues.  Once, the build and install finishes, I can go back to the KiCAD 5.1.12 upgrade work. I keep finding minor issues with my 5.1.12 upgrade working on 6.0.4.
I have decided I will not be able to fix 6.0.4 easily; so, I will wait till 6.0.5 and try again, to get it to work with my Intel GPU driver.

Do you think we should have two different KiCAD packages one for 6 and one for 5?

Tim S.

**Reply by revelator** [reply]
not sure, i use geforce 970 and gmsh works on that with opencascade-7.5.3 but can gmsh even be built using oce-0.18.3 ?.
installing both gives me conflicts as well and seems oce has some missing symbols present in opencascade.

**Reply by revelator** [reply]
also im having some problems building kicad when both wxwidgets 3.0.5 and 3.1.5 are installed, kicad only detects 3.0.5 and refuses to build unless i replace wx-config with wx-config-3.1 :/

**Comment by stahta01** [comment]
I am building KiCAD 5 with 3.0.x and KiCAD 6 with 3.1.x; I am once more testing my KiCAD 5 changes and will remove the WIP in a day or two. I will do an PR for KiCAD 6 in mingw-packages-dev as kicad-git.

Tim S.

**Reply by revelator** [reply]
ah ok sounds good then :)
